src/main/java/Duke.java

```diff
@@ -1,10 +1,35 @@
+import java.io.FileNotFoundException;
```

Comment:
Really good & clean implementation! Couldn't find much problems.
Well done! :-) 

Hope to learn a lot from you.

Cheers!

Reply:
Thank you :)